src/cc65/stdfunc.c

```diff
@@ -284,7 +284,7 @@ static void StdFunc_memcpy (FuncDesc* F attribute ((unused)), ExprDesc* Expr)
             Label = GetLocalLabel ();
 
             /* Generate memcpy code */
-            if (Arg3.Expr.IVal <= 127) {
+            if (Arg3.Expr.IVal <= 129) {
```

Comment:
These conditions are for loops like:

         ldy #IVal-1
    loop ...
         dey
         bpl loop

They will work for `ldy #128` and lower, hence IVal<=129.